draft-ietf-quic-http.md

```diff
@@ -536,6 +536,11 @@ not acting as a tunnel) MUST NOT forward a malformed request or response.
 Malformed requests or responses that are detected MUST be treated as a stream
 error ({{errors}}) of type H3_GENERAL_PROTOCOL_ERROR.
 
+A tunnel that dispatches HTTP messages to different endpoints might forward a
```

Comment:
```suggestion
An intermediary acting as a tunnel could carry HTTP messages from multiple endpoints on one side to a single endpoint on the other side within a single HTTP/3 connection. Such an intermediary might forward malformed HTTP messages as well. To limit the scope of damage to the offending endpoint, an endpoint that detects a malformed request or response MUST NOT escalate this error to a  connection-level error unless specified otherwise.
```

Reply:
I think that this is a "cannot" rather than a "MUST NOT".  What that intermediary does is up to them.  It might be inadvisable to have errors from one source affect all of the sources that share the same connection, but that's their call.

Reply:
@janaiyengar Thank you for the suggestion. Applied.

@martinthomson Yeah, MUST NOT applies to what the endpoint does when it receives a malformed message from the peer which _might_ be an intermediary.